### DISCUSSION TITLE
Missing attributes in Vendor Purchase Order API response

### DISCUSSION THREAD

**Discussion by ANarayanan-Cervello** [amzn/selling-partner-api-models#609] [Ideas]
Hi Team,
              I am getting the following attributes at an item(ASIN) level from the API response for each purchase order using the endpoint (https://github.com/amzn/selling-partner-api-docs/blob/main/references/vendor-orders-api/vendorOrders.md#get-vendorordersv1purchaseorders) :

```
'itemSequenceNumber': '1',
'amazonProductIdentifier': 'xxxxxxxxx',
'vendorProductIdentifier': 'xxxxxxxxx',
'orderedQuantity': {'amount': x, 'unitOfMeasure': 'xxx', 'unitSize': x},
'isBackOrderAllowed': False,
'netCost': {'currencyCode': 'USD', 'amount': 'xxx'},
'listPrice': {'currencyCode': 'USD', 'amount': 'xxx'}}]}}

```
However, when I look for the purchase orders up on the Vendor Central, I can see additional information/attributes
namely-> 'Expected Date', 'Accepted Quantity', 'Quantity received',
'Quantity Outstanding'.

I understand that details like expected date , accepted quantity quantity and quantity received/outstanding are shipment related information. However, why don't we have these added to the existing `get-vendorordersv1purchaseorders` endpoint, so that the end result mirrors  the report that is generated from the portal . Alternatively, we could have a new GET endpoint under Shipments (`vendor-shipments-api/vendorShipments.md`) , wherein we could retrieve shipment related information by passing in a purchase order number. Thoughts?

**Comment by ANarayanan-Cervello** [comment]
@chapmanjw Is there going to be any work for this?

**Comment by NA-Dev** [comment]
I would also benefit from being able fetch Expected Date, Accepted Qty, and Quantity Received from the API.

**Comment by MeenaAmz** [comment]
I suggest that you take a look at [Get purchase order status ](https://developer-docs.amazon.com/sp-api/docs/vendor-orders-api-v1-reference#get-vendorordersv1purchaseordersstatus) This one has details on the quantity received (Payload_orderstatus_itemstatus_receivingstatus) and acknowledged (payload_ordersstatus_itemstatus_acknowledgementstatus). [Use case guide](https://developer-docs.amazon.com/sp-api/docs/vendor-retail-procurement-apis-v1-use-case-guide#purchase-order-and-acknowledgement-use-cases) has a few sample use cases
